lib/process-services-cloud/src/lib/task/task-list/components/service-task-list-cloud.component.spec.ts

```diff
@@ -467,50 +429,44 @@ describe('ServiceTaskListCloudComponent', () => {
             });
             fixture = TestBed.createComponent(ServiceTaskListCloudComponent);
             component = fixture.componentInstance;
-            element = fixture.debugElement.nativeElement;
             taskSpy = spyOn(serviceTaskListCloudService, 'getServiceTaskByRequest').and.returnValue(of(fakeServiceTask));
 
         });
         afterEach(() => {
             fixture.destroy();
         });
 
-        it('shoud show tooltip if config copyContent flag is true', fakeAsync(() => {
+        it('shoud show tooltip if config copyContent flag is true', () => {
             taskSpy.and.returnValue(of(fakeServiceTask));
-            const appName = new SimpleChange(null, 'FAKE-APP-NAME', true);
+            component.presetColumn = 'fakeCustomSchema';
 
-            component.success.subscribe(() => {
-                fixture.whenStable().then(() => {
-                    fixture.detectChanges();
-                    const spanHTMLElement = element.querySelector<HTMLInputElement>('span[title="04fdf69f-4ddd-48ab-9563-da776c9b163c"]');
-                    spanHTMLElement.dispatchEvent(new Event('mouseenter'));
-                    fixture.detectChanges();
-                    expect(fixture.debugElement.nativeElement.querySelector('.adf-copy-tooltip')).not.toBeNull();
-                });
-            });
+            component.reload();
+            fixture.detectChanges();
```

Comment:
This test is not testing anymore a lot of stuff that was removed on success. why?

Reply:
:D maybe a little bit more of refactoring to make the test more clear would be a plus